spec/lib/data_migration_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'data_migration'
+
+RSpec.describe DataMigration do
+  let!(:unmigrated_asset) { FactoryGirl.create(:asset, uuid: 'some-uuid') }
+  let!(:migrated_asset) { FactoryGirl.create(:asset) }
```

Comment:
Should these names be reversed (i.e. unmigrated has a blank uuid and migrated has a real uuid)?